Subject: drop wasm-bindgen-test

The "version" crate isn't available in Debian, and we're not targeting
wasm closely, so it's simplest to drop this for now.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -154,9 +154,6 @@
 version = "0.2"
 optional = true
 
-[target.'cfg(all(target_arch = "wasm32", not(any(target_os = "emscripten", target_os = "wasi"))))'.dev-dependencies.wasm-bindgen-test]
-version = "0.3"
-
 [target.'cfg(target_os = "android")'.dependencies.android-tzdata]
 version = "0.1.1"
 optional = true
